CHANGELOG.md

```diff
@@ -19,6 +19,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - The tool, currently under the `tools` directory, was originally named `oecert` under the `tests/tools` directory.
   - The tool can be used to generate certificates, reports, and evidence in various formats.
   - The tool is for debugging purposes and is not suitable for production use.
+- Add the CapturePFGPExceptions configuration for SGX enclaves, which allows opting into the SGX2 feature of capturing #PF/#GP exceptions inside an enclave.
+  - Developers can specify the CapturePFGPExceptions with a binary value in the enclave config file or set the value via the newly added OE_SET_ENCLAVE_SGX2 m
+acro, which is used to set SGX2-specific properties.
+  - After opting into the feature (value=1), the in-enclave exception handler can capture the #PF (with the OE_EXCEPTION_PAGE_FAULT code) and #GP (with the co
+de OE_EXCEPTION_ACCESS_VIOLATION) exceptions.
+  - More information about the exceptions can be found in the `faulting_address` and `error_code` members of the `oe_exception_record_t` structure passed into
+ the handler.
+  - The feature is only available on SGX2 CPUs and is ignored on SGX1 CPUs.
```

Comment:
What does it mean for a feature to be ignored? (as opposed to not available)

If you meant to say that the mechanism to _opt-in_ is ignored, then that's not a great developer experience to have a developer opt in and have nothing happen.   It's usually better to fail the opt-in than to not adhere to a contract.  The alternative would be to rename the opt-in to be a _preference_ rather than a directive.

Reply:
Updated the text, can you check again?